src/tests/multihost/ad/test_cifs.py

```diff
@@ -14,19 +20,19 @@ class Testcifs(object):
     @pytest.mark.tier1
     def test_0001_wbinfo(self, multihost):
         """
-        @Title: IDM-SSSD-TC: samba_idmap: Samba can not
-        register sss idmap module due to SMB_IDMAP_INTERFACE_VERSION
-
-        @Setup:
-        1. Join RHEL system to windows AD domain using below command
-           $ realm join -v TESTRELM.TEST --membership-software=samba
-        2. configure smb.conf to use sss as idmap backend
-           idmap config * : backend = sss
-           idmap config * : range   = 200000-2147483647
-        3. Restart winbind
-
-        @Steps:
-        1. Run wbinfo -i <DOMAIN>\administrator
+        :title: IDM-SSSD-TC: samba_idmap: Samba can not
+         register sss idmap module due to SMB_IDMAP_INTERFACE_VERSION
+        :id: bad2770e-75de-4b41-af47-e9a38b8c0e73
+        :requirement: IDM-SSSD-REQ: Samba with sssd as idmap backend
+        :setup:
```

Comment:
I think "setup" should be part of test class not test function as all test functions contains same set of class fixture .

Reply:
Here we can put set up part within steps .

Reply:
I have moved the setup part to class level for now. If needed we will have to  update it per test by adding to steps if required.